### DISCUSSION TITLE
Gaming inclination on tdf bikes

### DISCUSSION THREAD

**Discussion by ibbr0** [cagnulein/qdomyos-zwift#672] [Ideas]
I’m riding a proform tdf1.0 with zwift thanks to qz. Everything works, qz controls inclination and/or resistance as desired. My one beef with the bike  has always been that it is hard to get much resistance unless the inclination is steep. I could see the possibility of qz providing an inclination factor to multiply inclination commands sent to the bike. If set to 1 it would just send actual inclination. So auto inclination would still work, but could be set to make climbs harder if desired.

**Comment by cagnulein** [comment]
@ibbr0 i added this to the beta already! send me an email to roberto.viola83 at gmail.com indicating this and if you are on ios or android and i will send you the beta!
